### PR TITLE
#48 more declarative tests

### PR DESCRIPTION
**Pull request by Vatavuk**
Refactored test to be more declarative. Issue #48

### CODE REVIEW

File: src/test/java/org/cactoos/http/HtHeadTest.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import java.util.Random;
 import org.cactoos.io.BytesOf;
 import org.cactoos.io.InputOf;
+import org.cactoos.matchers.TextHasString;
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtHeadTest.java]: @Vatavuk please use `org.llorllale.cactoos.matchers.TextHasString` (we'll soon be removing `org.cactoos.matchers` from the classpath)

File: src/test/java/org/cactoos/http/io/SkipInputTest.java [hunk 2]
```diff
@@ -26,10 +26,10 @@
 
 import org.cactoos.io.BytesOf;
 import org.cactoos.io.InputOf;
+import org.cactoos.matchers.TextHasString;
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/io/SkipInputTest.java]: @Vatavuk please use `org.llorllale.cactoos.matchers.TextHasString` (we'll soon be removing `org.cactoos.matchers` from the classpath)

File: src/test/java/org/cactoos/http/HtBodyTest.java [hunk 3]
```diff
@@ -23,12 +23,11 @@
  */
 package org.cactoos.http;
 
-import java.io.IOException;
 import org.cactoos.io.InputOf;
+import org.cactoos.matchers.TextHasString;
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtBodyTest.java]: @Vatavuk please use `org.llorllale.cactoos.matchers.TextHasString` (we'll soon be removing `org.cactoos.matchers` from the classpath)

**Comment by llorllale** [comment]
@Vatavuk see above

Also, there is a conflict on `HtCookiesTest`

**Comment by Vatavuk** [comment]
@llorllale fixed

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos-http#58` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#58](https://github.com/yegor256/cactoos-http/issues/58) is now out of scope

**Pull request merged**